pkgs/by-name/la/lazyjj/package.nix

```
@@ -0,0 +1,47 @@
+{
+  lib,
+  binutils-unwrapped,
+  fetchFromGitHub,
+  makeWrapper,
+  jujutsu,
+  rustPlatform,
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "lazyjj";
+  version = "0.3.1";
+
+  src = fetchFromGitHub {
+    owner = "Cretezy";
+    repo = "lazyjj";
+    rev = "v${version}";
+    hash = "sha256-VlGmOdF/XsrZ/9vQ14UuK96LIK8NIkPZk4G4mbS8brg=";
+  };
+
+  cargoHash = "sha256-TAq9FufGsNVsmqCE41REltYRSSLihWJwTMoj0bTxdFc=";
+
+  postInstall = ''
+    # mv $out/bin/lazyjj $out/bin/lazyjj-unwrapped
+    wrapProgram $out/bin/lazyjj \
+      --prefix PATH : ${
+        lib.makeBinPath [
+          binutils-unwrapped
+          jujutsu
+        ]
+      }
+  '';
+
+  nativeBuildInputs = [ makeWrapper ];
+
+  nativeCheckInputs = [
+    binutils-unwrapped
+    jujutsu
+  ];
+
+  meta = with lib; {
+    description = "TUI for Jujutsu/jj";
+    homepage = "https://github.com/Cretezy/lazyjj";
+    license = licenses.asl20;
```

issue: wrong license

```suggestion
    license = licenses.apache_2_0;
```